service/controller/legacy/controller.go

```diff
@@ -33,6 +33,7 @@ type ControllerConfig struct {
 	Provider                string
 	Region                  string
 	Registry                string
+	IngressApiVersion       string
```

Comment:
In Go we usually capitalize abbreviations so this should be `IngressAPIVersion`. See `AlertmanagerCreatePVC` below for example.